pkg/runtest/run_test.go

```diff
@@ -93,7 +77,7 @@ func test(t *testing.T, sysTarget *targets.Target) {
 		Dir:          filepath.Join("..", "..", "sys", target.OS, targets.TestOS),
 		Target:       target,
 		Tests:        *flagFilter,
-		Features:     features.ToFlatRPC(),
+		Features:     0,
```

Comment:
pkg/runtest tests run only for "test" OS which does not have any feature nor custom setup.
We can't run any other OSes as normal Go tests.
So instead I just dropped that unneeded code instead of porting it (too many tools that duplicate code and need to be ported).